portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,105 @@
+body {
+    background-color: black;
+    color: white;
+    margin-left: 40px;
+    margin-right: 20px;
+}
+
+h1{
+    font-size: 75px;
+    font-weight: 700;
+    font-family: cursive;
+    margin-left: 16%;
```

Comment:
nit: order the rules alphabetically. It makes it easy to quickly find properties within a rule.
font-family
font-size
font-weight
margin

Reply:
resolved